### PR TITLE
Community Meeting is at 1800UTC

### PR DESCRIPTION
**Pull request by gundalow**

### CODE REVIEW

File: ansible_community_meetings.ics [hunk 1]
```diff
@@ -5,8 +5,6 @@ BEGIN:VEVENT
 SUMMARY:Ansible Security Automation working group
 DTSTART;VALUE=DATE-TIME:20210405T150000Z
 DURATION:PT1H
-DTSTAMP;VALUE=DATE-TIME:20191126T180022Z
```
Comment (gundalow) [inline ansible_community_meetings.ics:8]: These lines seem to have been added in the previous commit, though don't appear in previous versions.

File: meetings/README.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ You can add this full list of meetings to your personal calendar by importing by
   ([ical](https://raw.githubusercontent.com/ansible/community/main/meetings/ical/network.ics))
   in `#ansible-network`
 
-* [19:00 UTC](http://www.thetimezoneconverter.com/?t=19:00&tz=UTC):
+* [16:00 UTC](http://www.thetimezoneconverter.com/?t=16:00&tz=UTC):
```
Comment (felixfontein) [inline meetings/README.md:56]:
```suggestion
* [18:00 UTC](http://www.thetimezoneconverter.com/?t=18:00&tz=UTC):
```

Comment (Kogelvis) [inline meetings/README.md:56]: Shouldn't this be 18 instead of 16?

File: ansible_community_meetings.ics (hunk 1 above)
Comment (felixfontein) [reply ansible_community_meetings.ics:8]: Maybe another version of the tool that generates the file?

File: meetings/README.md (hunk 2 above)
Comment (gundalow) [reply meetings/README.md]: Yup, good spot, thanks

File: ansible_community_meetings.ics (hunk 1 above)
Comment (gundalow) [reply ansible_community_meetings.ics:8]: I updated yaml2ics and I didn't get them. Maybe a mac vs Linux thing?

**Pull request merged**